src/Test/Perf/Runner/Perf.Runner.csproj

```diff
@@ -80,4 +80,10 @@
     <Folder Include="CPC-Consumption\" />
   </ItemGroup>
   <Import Project="..\..\..\..\build\Targets\VSL.Imports.targets" />
+  <PropertyGroup>
+    <PostBuildEvent>call "$(MSBuildThisFileDirectory)install_benchview_tools.cmd" "$(OutDir)"</PostBuildEvent>
```

Comment:
We shouldn't be installing tools as a part of a post build step.  That will potentially have a pretty negative impact on our ability to build and F5 inside of Visual Studio.  This logic should be moved to the scenarios in which the bench view tools are run.